internal/config/validation.go

```diff
@@ -24,22 +23,27 @@ import (
 	"github.com/edgelesssys/constellation/v2/internal/versionsapi"
 	ut "github.com/go-playground/universal-translator"
 	"github.com/go-playground/validator/v10"
-	"go.uber.org/multierr"
 	"golang.org/x/mod/semver"
 )
 
-// DisplayValidationErrors shows all validation errors inside configError as one formatted string.
-func DisplayValidationErrors(errWriter io.Writer, configError error) error {
-	errs := multierr.Errors(configError)
-	if errs != nil {
-		fmt.Fprintln(errWriter, "Problems validating config file:")
-		for _, err := range errs {
-			fmt.Fprintln(errWriter, "\t"+err.Error())
-		}
-		fmt.Fprintln(errWriter, "Fix the invalid entries or generate a new configuration using `constellation config generate`")
-		return errors.New("invalid configuration")
+// ValidationError occurs when the validation of a config fails.
+// It contains a list of errors that occurred during validation.
+type ValidationError struct {
+	validationErrMsgs []string
+}
+
+func (e *ValidationError) Error() string {
+	return "invalid configuration"
+}
+
+// LongMessage prints the errors that occurred during validation in a verbose and user friendly way.
+func (e *ValidationError) LongMessage() string {
+	msg := "Problems validating config file:\n"
+	for _, ve := range e.validationErrMsgs {
+		msg += fmt.Sprintf("\t%s\n", ve)
 	}
-	return nil
+	msg += fmt.Sprintln("Fix the invalid entries or generate a new configuration using `constellation config generate`")
```

Comment:
```suggestion
	msg += "Fix the invalid entries or generate a new configuration using `constellation config generate`\n"
```
Don't think we need `fmt.Sprintln()` here, no?

Reply:
Hm, maybe this shouldn't end this with a newline anyway? String() like functions normally don't end strings with a newline, and we already print this message with PrintErrln...

Reply:
In any case we probably don't need to wrap the string with `Sprintln`